games/x.yaml

```diff
@@ -83,6 +83,21 @@
   updated: 2020-12-16
   url: http://xmoto.tuxfamily.org/
 
+- name: X-Settlers
+  development: active
+  images:
+  - https://raw.githubusercontent.com/Skarmux/X-Settlers/master/xsettlers/docs/screenshots/xsettlers_textured.png
+  lang:
+  - C++
+  license:
+  - Apache-2.0
```

Comment:
The `Apache-2.0` license doesn't exist in the validation schema: https://github.com/opengaming/osgameclones/blob/master/schema/games.yaml#L110

I think you could just use `Apache` in this case, what do you think?

Reply:
Yes, fixed it.